sdk/identity/identity/test/manual/src/index.tsx

```diff
@@ -4,205 +4,350 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 
-import { InteractiveBrowserCredential, BrowserLoginStyle } from "@azure/identity";
-import { KeyClient, KeyVaultKey } from "@azure/keyvault-keys";
-
-const getRandomKeyName = () => `key${Math.random()}`.replace(/\./g, "");
+import {
+  InteractiveBrowserCredential,
+  BrowserLoginStyle,
+  InteractiveBrowserAuthenticationFlow,
+} from "@azure/identity";
+import { ServiceBusClient } from "@azure/service-bus";
 
 interface ClientDetails {
```

Comment:
One more error
![image](https://user-images.githubusercontent.com/10452642/107593084-5f80bf80-6bc3-11eb-804f-4595bb30bba0.png)

Reply:
Gone when tested with the artifact from #13263